libs/article/website/src/lib/article-list/article-list.tsx

```diff
@@ -1,5 +1,12 @@
-import {css, styled} from '@mui/material'
+import {Pagination, css, styled} from '@mui/material'
 import {BuilderArticleListProps, useWebsiteBuilder} from '@wepublish/website/builder'
+import {useEffect, useMemo, useState} from 'react'
+
+export const PaginationStyled = styled(Pagination)`
```

Comment:
No styled post fix please. **Ever** when it gets exported.

Reply:
Yes, but inside the website builder.

Reply:
It's part of the library but not the builder itself. So it can not be overridden. Checkout the WebsiteBuilderContext